iModelCore/Bentley/Tests/NonPublished/Cpp_Test.cpp

```diff
@@ -363,10 +363,10 @@ void test_random ()
     {
     // common use case: binding a RNG with a distribution
     std::default_random_engine e;
-    std::uniform_int_distribution<> d(0, 10);
-    std::function<int()> rnd = std::bind(d, e);
+    std::uniform_int_distribution<uint64_t> d(0, 10);
+    std::function<uint64_t()> rnd = std::bind(d, e);
     for(int n=0; n<10; ++n)
-        printf ("%d ", rnd());
+        printf ("%lld ", rnd());

```

Comment:
```suggestion
        printf ("%" PRIu64 " ", rnd());
```

`lld` is for signed values; your value is unsigned. `lld` is not portably 64 bits; the `PRI` macros specify the exact size.

Reply:
Thanks, I have fixed it now.